{"uuid": "8e21da5a-1e79-4397-8ab2-684af210bfa4", "children": ["13ddcf2f-8d67-400e-b443-7558b99719d0"], "befores": [{"name": "remote_driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: Could not start a new session. New session request timed out \nHost info: host: '1648e4e36e7d', ip: '172.18.0.3'\nBuild info: version: '4.14.1', revision: '03f8ede370'\nSystem info: os.name: 'Linux', os.arch: 'amd64', os.version: '6.5.0-1-MANJARO', java.version: '11.0.20.1'\nDriver info: driver.version: unknown\nStacktrace:\n    at org.openqa.selenium.grid.sessionqueue.local.LocalNewSessionQueue.addToQueue (LocalNewSessionQueue.java:224)\n    at org.openqa.selenium.grid.sessionqueue.NewSessionQueue.lambda$new$0 (NewSessionQueue.java:68)\n    at org.openqa.selenium.remote.http.Route$TemplatizedRoute.handle (Route.java:193)\n    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)\n    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:345)\n    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)\n    at org.openqa.selenium.grid.sessionqueue.NewSessionQueue.execute (NewSessionQueue.java:128)\n    at org.openqa.selenium.remote.tracing.SpanWrappedHttpHandler.execute (SpanWrappedHttpHandler.java:87)\n    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:63)\n    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:345)\n    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)\n    at org.openqa.selenium.grid.router.Router.execute (Router.java:87)\n    at org.openqa.selenium.grid.web.EnsureSpecCompliantResponseHeaders.lambda$apply$0 (EnsureSpecCompliantResponseHeaders.java:34)\n    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:63)\n    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:345)\n    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)\n    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:345)\n    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)\n    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:345)\n    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)\n    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:345)\n    at org.openqa.selenium.remote.http.Route.execute (Route.java:69)\n    at org.openqa.selenium.remote.AddWebDriverSpecHeaders.lambda$apply$0 (AddWebDriverSpecHeaders.java:35)\n    at org.openqa.selenium.remote.ErrorFilter.lambda$apply$0 (ErrorFilter.java:44)\n    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:63)\n    at org.openqa.selenium.remote.ErrorFilter.lambda$apply$0 (ErrorFilter.java:44)\n    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:63)\n    at org.openqa.selenium.netty.server.SeleniumHandler.lambda$channelRead0$0 (SeleniumHandler.java:44)\n    at java.util.concurrent.Executors$RunnableAdapter.call (Executors.java:515)\n    at java.util.concurrent.FutureTask.run (FutureTask.java:264)\n    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1128)\n    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:628)\n    at java.lang.Thread.run (Thread.java:829)\n", "trace": "  File \"/usr/local/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/app/conftest.py\", line 29, in remote_driver\n    driver = webdriver.Remote(\n             ^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 205, in __init__\n    self.start_session(capabilities)\n  File \"/usr/local/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 289, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/usr/local/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1698158225491, "stop": 1698158525500}], "start": 1698158225491, "stop": 1698158525544}